<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="Sex_group">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>性別</string>
    </property>
    <widget class="QRadioButton" name="male">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>男</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="female">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>女</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="favorite_food">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 繁黑體 Std B</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>美食大集</string>
    </property>
   </widget>
   <widget class="QLabel" name="Profession">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>職業</string>
    </property>
   </widget>
   <widget class="QListWidget" name="choose_profession">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>廚師</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>法師</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>墮仙人</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>劍士</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>魔女</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>一鳴驚 人</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>道人</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>書法家</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>精算師</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>商人</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>駝客</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>少爺</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>拳擊教練</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>運動家</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>工程師</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>教師</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>學生</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="type_name">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="main_food">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>最喜歡的主食</string>
    </property>
    <widget class="QRadioButton" name="rice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>98</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>飯</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="noodle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>98</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>麵</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="noodle_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>98</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>冬粉</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="cuisine">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>最喜歡哪一種料理</string>
    </property>
    <widget class="QRadioButton" name="east">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>98</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>中式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="west">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>98</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>西式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="JP">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>98</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>日式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Kor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>98</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>韓式</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>99</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="serious_or_not">
     <item>
      <widget class="QCheckBox" name="serious">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>我有認真寫問卷</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Enter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="show_name">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>91</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="show_sex">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>51</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="show_profession">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>91</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="show_main_food">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>91</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="show_cuisine">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>101</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>性別</string>
    </property>
   </widget>
   <widget class="QLabel" name="Profession_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>職業</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>主食</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>料理</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Garamond Pro</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Load">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>40</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Sun</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Mon</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Tue</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Wed</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Thu</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Fri</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Sat</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>早餐</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>中餐</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>晚餐</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="0" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="1">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="0">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="0">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="1">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="4" column="0">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="4" column="1">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="4" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="5" column="0">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="5" column="1">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="5" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="6" column="0">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="6" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="name_5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>20</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用餐時間(請點擊勾選)：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1320</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
